relay-event-normalization/src/normalize/span/description/mod.rs

```diff
@@ -76,6 +76,10 @@ pub(crate) fn scrub_span_description(span: &Span) -> Option<String> {
                 // be low-risk to start adding the description.
                 Some(description.to_owned())
             }
+            ("ui", sub) if sub.starts_with("interaction.") || sub.starts_with("react.") => data
+                .and_then(|data| data.get("ui.component_name"))
```

Comment:
Should we have a fallback in case `ui.component_name` doesn't exist?

Reply:
The fallback is `None`, so no description and no group hash. This is intentional. Good point though!